web-security/introduction-securite-web-pour-futurs-developpeurs-short.md

```diff
@@ -25,9 +25,9 @@ Mais aussi parce que réfléchir aux problématiques de la sécurité pour vos p
 
 --------------------------------------------------------------------------------
 
-# Amusant?
+# Amusant ?
 
-.notes: ...Oui
+.notes : ...Oui
```

Comment:
Not sure this is supported, here it's not a french spacing, that's a 'language feature' and it should stay an english spacing (no space)